run-tests.php

```diff
@@ -1439,8 +1437,7 @@ function run_all_tests_parallel(array $test_files, array $env, ?string $redir_te
             "GLOBALS" => $GLOBALS,
             "constants" => [
                 "INIT_DIR" => INIT_DIR,
-                "TEST_PHP_SRCDIR" => TEST_PHP_SRCDIR,
-                "TRAVIS_CI" => TRAVIS_CI
+                "TEST_PHP_SRCDIR" => TEST_PHP_SRCDIR
```

Comment:
Keep the trailing comma.

```suggestion
                "TEST_PHP_SRCDIR" => TEST_PHP_SRCDIR,
```

Reply:
Of course. The run-tests.php should work for the current version in the current branch. I thought it should work for some PHP 7.4 also or something. Ok.